### ISSUE TITLE
python 3 compatibility

### ISSUE THREAD

**Issue by sdyroff** [geier/pycarddav#122]
It seems like the last dependency vobject is now with version 0.9.0 python 3 compatible. Are there any plans on making pycarddav python 3 compatible? I would like to help with porting, but wanted to coordinate before starting to hack :)

**Comment by geier** [comment]
pycarddav development is currently on hold (from my side), when I find the time I want to remove the syncing part and use vdir's instead (see #92). But I'll gladly accept any pull requests (including python 3 support).

**Comment by sdyroff** [comment]
I created a branch [python3](https://github.com/sdyroff/pycarddav/tree/python3), that works for me with python3. I was able to do a

    # pycardsyncer
    # pc_query

with this modifications. Unfortunately I cannot run the testsuite at the moment. Maybe someone else can try my changes.
